### PR TITLE
buffs explosive holo trap

### PR DESCRIPTION
**Pull request by ShadowDeath6**
their traps last longer and they have a longer cooldown in between setting traps

as always vars adjusted are up for debate

:cl: ShadowDeath6
bugfix: Explosive holoparasite's bomb trap now actually works.
tweak: Increases the duration of the explosive holoparasite's bomb trap, as well as the cooldown between bomb traps set.
/:cl:

### CODE REVIEW

**Comment by ShadowDeath6** [comment]
rused

File: code/modules/mob/living/simple_animal/guardian/types/explosive.dm [hunk 1]
```diff
@@ -61,7 +62,7 @@
 	anchored = A.anchored
 	density = A.density
 	appearance = A.appearance
-	spawn(600)
+	spawn(1000)
```
Comment (Cruix) [inline code/modules/mob/living/simple_animal/guardian/types/explosive.dm]: You should make this use a timer instead of a spawn.

**Comment by ShadowDeath6** [comment]
DNM, actually, bomb holo traps are completely unresponsive. I'm just about finished with the fix, though

**Comment by ShadowDeath6** [comment]
r4m @Cruix @X-TheDark

**Pull request merged**